hw1/Sem01Task02.py

```diff
@@ -0,0 +1,12 @@
+# Задача 2: Найдите сумму цифр трехзначного числа.
+
+# спрашиваем у пользователя число (не обязательно трехзначное)
+number = int(input("Введите число: "))
+sum = 0  # инициализируем переменную под сумму
+
+while (number != 0):
+    sum = sum + number % 10  # прибавляем к сумме остаток от целочисленного делеения на 10
```

Comment:
Можно использовать быструю запись
```suggestion
    sum += number % 10  # прибавляем к сумме остаток от целочисленного делеения на 10
```